types/pg/index.d.ts

```diff
@@ -73,7 +73,7 @@ export declare class Pool extends events.EventEmitter {
     end(): Promise<void>;
 
     query(queryText: string, values?: any[]): Promise<QueryResult>;
-
+    query(queryText: QueryConfig, callback: (err: Error, result: QueryResult) => void): void;
     query(queryText: string, callback: (err: Error, result: QueryResult) => void): void;
```

Comment:
Thanks, I'll address this. Looking at the spec for client queries it's missing another couple of bits too so I'll fix it properly.